appmanifest/shortcuts-member/shortcuts-member-skip-for-invalid-url.webmanifest

```diff
@@ -0,0 +1,17 @@
+{
+    "shortcuts": [{
```

Comment:
As above... making sure we don't get a false positive or a false fail because of missing manifest members that Edge likes.